pkg/secrets/b64/manager.go

```diff
@@ -30,6 +32,6 @@ func NewBase64SecretsManager() secrets.Manager {
 type manager struct{}
 
 func (m *manager) Type() string                         { return Type }
-func (m *manager) State() interface{}                   { return map[string]string{} }
+func (m *manager) State() json.RawMessage               { return nil }
```

Comment:
This is a difference in behavior.
`json.RawMessage(nil)` is empty.
`json.Marshal(map[string]string{})` is `{}`.

This should probably match.
```suggestion
func (m *manager) State() json.RawMessage {
  return json.RawMessage("{}")
}
```

Reply:
This was intentional. It's ok to return nil for state if there's no state to save.